assets/training/finetune_acft_hf_nlp/components/pipeline_components/chat_completion/spec.yaml

```diff
@@ -627,30 +627,30 @@ jobs:
       # converted_model: '${{parent.jobs.chat_completion_finetune.outputs.mlflow_model_folder}}'
     outputs:
       mlflow_model_folder: '${{parent.outputs.mlflow_model_folder}}'
-  # model_prediction:
-  #   type: command
-  #   component: azureml:model_prediction_with_container:0.0.2
-  #   compute: '${{parent.inputs.compute_model_evaluation}}'
-  #   resources:
-  #     instance_type: '${{parent.inputs.instance_type_model_evaluation}}'
-  #   inputs:
-  #     task: chat-completion
-  #     test_data: '${{parent.jobs.chat_completion_datapreprocess.outputs.output_dir}}'
-  #     label_column_name: messages
-  #     mlflow_model: '${{parent.jobs.ft_nlp_model_converter.outputs.mlflow_model_folder}}'
-  #     evaluation_config_params: '${{parent.inputs.evaluation_config_params}}'
-  # compute_metrics:
-  #   type: command
-  #   component: azureml:compute_metrics:0.0.31
-  #   compute: '${{parent.inputs.compute_model_evaluation}}'
-  #   resources:
-  #     instance_type: '${{parent.inputs.instance_type_model_evaluation}}'
-  #   inputs:
-  #     task: chat-completion
-  #     ground_truth: '${{parent.jobs.model_prediction.outputs.ground_truth}}'
-  #     prediction: '${{parent.jobs.model_prediction.outputs.predictions}}'
-  #     prediction_probabilities: '${{parent.jobs.model_prediction.outputs.prediction_probabilities}}'
-  #     evaluation_config: '${{parent.inputs.evaluation_config}}'
-  #     evaluation_config_params: '${{parent.inputs.evaluation_config_params}}'
-  #   outputs:
-  #     evaluation_result: '${{parent.outputs.evaluation_result}}'
+  model_prediction:
+    type: command
+    component: azureml:model_prediction_with_container:0.0.5
+    compute: '${{parent.inputs.compute_model_evaluation}}'
+    resources:
+      instance_type: '${{parent.inputs.instance_type_model_evaluation}}'
+    inputs:
+      task: chat-completion
+      test_data: '${{parent.jobs.chat_completion_datapreprocess.outputs.output_dir}}'
+      label_column_name: messages
+      mlflow_model: '${{parent.jobs.ft_nlp_model_converter.outputs.mlflow_model_folder}}'
+      evaluation_config_params: '${{parent.inputs.evaluation_config_params}}'
+  compute_metrics:
+    type: command
+    component: azureml:compute_metrics:0.0.31
```

Comment:
Compute metrics latest component is 0.0.35

Reply:
Thanks, fixed